lib/dish.rb

```diff
@@ -0,0 +1,32 @@
+class Dish
+  def initialize
+    @dishes = {
+      pizza: '£6.99',
+      spaghetti: '£7.20',
+      lasagne: '£7.80',
+      chips: '£2.40',
+      tiramisu: '£4.20'
+    }
+    @selection = {}
+  end
+
+  def list_items
+    @dishes
+  end
+
+  def select_dishes(dish_name, dish_quantity)
+    @selection[dish_name] = dish_quantity
+    @selection
+  end
+
```

Comment:
Succinct use of hashes - did you find having this format for the data helpful later on?